src/components/FreeSoftware.tsx

```diff
@@ -10,11 +10,21 @@ export default function FreeSoftware() {
           <div className="col col--6">
             <Svg className={styles.logo} />
             <h1>Your media, your server, your way</h1>
-            <div className="margin-bottom--sm"><b>Jellyfin is fully self-hosted and fully open source.</b></div>
-            <div>We don't do tracking or paid plans. There is no forced connection to a remote server. You are in control from start to finish.</div>
+            <div className="margin-bottom--sm">
+              <b>Jellyfin is fully self-hosted and fully open source.</b>
+            </div>
+            <div>
+              We don&apos;t do tracking or paid plans. There is no forced
```

Comment:
Is this somehow more web compliant than a regular apostrophe?

Reply:
The React rules were complaining about using a normal single quote here, and recommended this as a replacement.